docs/content/getting-started.md

```diff
@@ -4,8 +4,8 @@ title: Getting started
 
 # Getting started
 
-HyperShift is middleware for hosting OpenShift control planes at scale that
-solves for cost and time to provision, as well as portability cross cloud with
```

Comment:
s/across cloud/across clouds/

Reply:
I think we should use `cloud service providers` instead of `cloud providers`